packages/docusaurus-theme-openapi/src/theme/ApiDemoPanel/ParamOptions/index.tsx

```diff
@@ -184,7 +190,7 @@ function ParamArrayFormItem({ param }: ParamProps) {
         value: values.length > 0 ? values : undefined,
       })
     );
-  }, [dispatch, items, param]);
+  }, [dispatch, items]);
```

Comment:
The way this was written before there is an infinite render loop.
Removing `param` from the dependency array fixed the infinite loop, but doing this introduces an eslint error. I'm not a react expert, so not really sure where to go next with this one. I'd be really open to feedback on how to handle this better.

Reply:
Superseded by d616c10de8ff23c9750c9ad477431c775faf7da4